maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -173,7 +177,17 @@ public CLIManager() {
                 .build());
         options.addOption(Option.builder(Character.toString(BATCH_MODE))
                 .longOpt("batch-mode")
-                .desc("Run in non-interactive (batch) mode (disables output color)")
+                .desc(
+                        "Run in non-interactive (batch) mode (disables output color) - alias for --non-interactive (kept for backwards compatability)")
+                .build());
+        options.addOption(Option.builder(NON_INTERACTIVE)
+                .longOpt("non-interactive")
+                .desc("Run in non-interactive (batch) mode (disables output color) - alias for --batch-mode")
+                .build());
+        options.addOption(Option.builder(FORCE_INTERACTIVE)
+                .longOpt("force-interactive")
+                .desc(
+                        "Run in interactive mode - even when the environment variable CI is set to true and --non-interactive or --batch-mode are set")
```

Comment:
Here we need to check that Jansi and friends get this flag properly since Jansi uses `isatty(2)` to test that.

Reply:
I'm not sure how to pass this information to Jansi. `isatty(2)` will atm just return 1/true. Also if `--batch-mode` is set. 
Do you think a change like this should be part of this PR?

Reply:
I'm not sure the flag is relevant for jansi.